metadata/expireFilesMetadata/README.md

```diff
@@ -0,0 +1,74 @@
+Artifactory Expires Files Metadata User Plugin
+==============================================
+
+This plugin expires files when they are requested according to file pattern and expire delay stored in properties file so that the files are redownloaded.
+
+This plugin is inspired from [Expires Packages Metadata User Plugin](https://github.com/jfrog/artifactory-user-plugins/tree/master/metadata/expirePackagesMetadata).
+
+`expireFilesMetadata.json`
+----------
+The json contains the properties for expiring delay for different repositories.
+
+JSon file format is Map where:
+- the key is the repository key of the remote repository where the following conf is applied
+  - delay : (in seconds) expire delay to force the download of the Files
+  - patterns : file patterns to apply thexpireFilesMetadata.jsonire delay for the remote repository
```

Comment:
Typo/copypaste error?

Reply:
Hi @DarthFennec,

thank you for your reviews and suggestions.

yep it was that top/copypaste.. 
I updated the README.md file and also add code to manage JSon data for HTTP POST method.
So that, I documented at the end section with classic 'curl test'.

Yes, I signed Jfrog CLA.

Regards